doc/make/index.md

```diff
@@ -45,6 +53,13 @@ This will compile the Lean library and binary into the `stage1` subfolder; see
 below for details. Add `-jN` for an appropriate `N` to `make` for a parallel
 build.
 
+To install the build run the following:
```

Comment:
I believe one reason this wasn't mentioned previously is that `elan toolchain link` (see below) mostly obsoletes the need for a global installation. Users had recurrent issues with unclean or duplicate global installations, so we might not want to advertise this option. Not sure what's more convenient on Windows.

Reply:
Ah, right, I'm not having much luck with `elan` in an msys2 environment.  Has that been tested?  When I ran all those `elan toolchain link ...` setup steps from `index.md`, typing `lean` yields `-bash: lean: command not found`

Reply:
Found the problem, the path "elan" prints out is wrong for msys2.  Probably should fix that somehow, but I added the right path information to msys2.md and remove the make install notes from ubuntu.md.